scripts/quests/windurst/Toraimarai_Turmoil.lua

```diff
@@ -81,9 +76,38 @@ quest.sections =
                         player:confirmTrade()
                         -- From previous implementation, award 100 fame (50 + 50) on first completion,
                         -- and 50 fame for any subsequent trade.
-                        if player:getQuestStatus(quest.areaId, quest.questId) == QUEST_ACCEPTED then
-                            player:addFame(xi.quest.fame_area.WINDURST, 50)
-                        end
+                        player:addFame(xi.quest.fame_area.WINDURST, 50)
```

Comment:
Can make this 100 fame now that it's not doubling up.

Reply:
it's still doubling up tho right because of line 21? i'm assuming that fame is awarded each time `quest:complete` is called

Reply:
Good question -- I can't review the full code from my phone, but:
1. The quest reward should be 100 fame, and that only gets called on quest:complete()
2.  Quest:complete() should only ever be called one time, and it changes the status from accepted to complete.  I'm not sure if there's any wonkiness to having it called repeatedly, but in the interest of keeping things tidy, I think we are better off with just using a quest_complete section and adding the 50 fame manually.